tmt/steps/provision/mrack.py

```diff
@@ -508,6 +515,7 @@ class GuestBeaker(tmt.steps.provision.GuestSsh):
     _data_class = BeakerGuestData
 
     # Guest request properties
+    whiteboard: str
```

Comment:
`whiteboard` in `BeakerGuestData` is annotated as `Optional[str]`, this `whiteboard` needs to match the annotation.

Reply:
Thanks for review. It's updated :-)